_drafts/Lab320Workbook.mdx

```diff
@@ -0,0 +1,287 @@
+# Lab 320 Workbook
+
+---
+
+In this lab, you will use the new **Storefront Builder** to make changes and additions to your company's Adobe Commerce Storefront powered by Edge Delivery Services. You will take the role of a marketing professional whose daily job involves optimizing the storefront for customer conversions and creating new product pages, marketing campaigns, and customer conversion experiments.
+
+## Lab Scenario
+
+Apple is releasing its latest iPhone. Your company, Citisignal, wants to launch a catalog update and promotional campaign related to this event. The goal is to increase sales and customer conversions. 
+
+## Lab Goals
+
+In this lab, you will learn how to:
+
+**Lesson 1:** Create a unique product details page <br/>
+**Lesson 2:** Run a promotional campaign targeted at general and loyal customers<br/>
+**Lesson 3:** Experiment with customer conversions in the cart.
+
+## Lab Warmup
+
+Make sure these things are working before you start the lab:
+
+- Log in to [https://da.live](https://da.live) with your provided account.
+- Open your project at: `https://da.live/#/adobe-summit-l320/seat-{{YOURSEAT}}`
+- You have the **Storefront Builder** (Code Name: Document Authoring) open in your browser.
+- You can view files and folders from your assigned `seat-xxx` storefront site content.
+- You can preview the landing page of your CitiSignal site by selecting the `index` file and opening the **Instant Preview** panel on the right.
```

Comment:
- Each document in the folder structure corresponds to a website page. You can open and edit it like you normally would with a document file
- You can see all the changes you make to the page live by opening the **Instant Preview** panel on the right.
- You can revert changes in case you made a mistake by using **Undo the last change** in the menu or Cmd+Z on your keyboard